enterprise_subsidy/apps/subsidy/tests/test_api.py

```diff
@@ -103,11 +103,11 @@ def test_existing_transaction_with_reversal(self):
             lms_user_id=self.lms_user_id,
             content_key=self.content_key
         )
-        reversal = Reversal.objects.create(
+        Reversal.objects.create(
             transaction=existing_transaction,
             idempotency_key=str(existing_transaction.idempotency_key) + '-reversed',
             quantity=19998,
-            state=TransactionStateChoices.COMMITTED,
+            state=TransactionStateChoices.CREATED,
```

Comment:
Reversal objects seem to be created with `state=CREATED`, so updating this to reflect that reality.

Reply:
If the steady state for a reversal is state=CREATED, then that is a bug.   I recommend that we leave this as it was, and I can pick up a fix in a separate PR.